frontend/src/css/subject.less

```diff
@@ -0,0 +1,49 @@
+.subject-content {
```

Comment:
Tend to think that part of this should be component specific. And part of it should be generalised.

I set a bad precedent at the start with card.less but haven't really used at all since, and have been trying to stop.

Reply:
I tried to use cards initially but it was way too inflexible so I went with this... The styling is the same for subject list and course list so I just used a less file.
It could be a 'FeedForSubjectsOrCourses' component instead but idk

Reply:
Comments below are about splitting these in any case.

Not sure what you mean by inflexible?

Reply:
Umm the grid layout stuff etc for cards... I wasn't sure how to make that do what I wanted.

Reply:
all g.